src/app/fdctl/run/tiles/fd_gossip.c

```diff
@@ -35,6 +35,7 @@
 #define PACK_OUT_IDX    2
 #define SIGN_OUT_IDX    3
 #define REPLAY_OUT_IDX  4
+#define VOTE_CONTACT_OUT_IDX    5
```

Comment:
this should just be contact out, not really specific to vote

Reply:
Hmm, it's actually true that the contact info sent out through this link is specific to TPU vote.

That's why you could find `ele->contact_info.tpu_vote.addr` and `ele->contact_info.tpu_vote.port` here: https://github.com/firedancer-io/firedancer/pull/2254/files#diff-4990d28d56483a6cad7d22a6dd0d2a4fd5c14196dfa762c00176b62e7aaf4a83R494